py/nightwatch/run.py

```diff
@@ -455,11 +461,24 @@ def make_plots(infile, basedir, preprocdir=None, logdir=None, rawdir=None, camer
     if (preprocdir is not None):
         #- plot preprocessed images
         downsample = 4
-
-        for camera in cameras:
-            input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits".format(camera, expid))
-            output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html".format(camera, expid))
-            web_plotimage.write_image_html(input, output, downsample, night)
+        
+        ncpu = get_ncpu(None)
+        input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits")
+        output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html")
+        
+        argslist = [(input.format(cam, expid), output.format(cam, expid), downsample, night) for cam in cameras]
+    
+        if ncpu > 1:
+            pool = mp.Pool(ncpu)
+            pool.starmap(web_plotimage.write_image_html, argslist)
+            pool.close()
+            pool.join()
+            
+        else:
+            for camera in cameras:
+                input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits".format(camera, expid))
+                output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html".format(camera, expid))
+                web_plotimage.write_image_html(input, output, downsample, night)
```

Comment:
The missing guide file changes look fine, but see #154 for suggested changes for this section.  Once that is updated and merged, please either rebase this or make same change here, then ok to merge this PR too.